tableau_utilities/scripts/datasource.py

```diff
@@ -11,6 +11,10 @@
 from tableau_utilities.tableau_server.tableau_server import TableauServer
 
 
+# Define color and symbol as globals
+color = Color()
+symbol = Symbol()
```

Comment:
```suggestion
COLOR = Color()
SYMBOL = Symbol()
```
Same as above comment